### PR TITLE
Add ability to request for feedback.

### PR DESCRIPTION
**Pull request by reduz**
Create template for requests for feedback.

### CODE REVIEW

**Review by aaronfranke — APPROVED** [review]
The existence of this form is a good idea, it would be a better fit for some kinds of issues, including those opened by the core devs.

Ideally, the two forms should automatically add labels to these issues. Before we can do this, we need to actually create the labels. A set of labels should probably be proposed/created by @akien-mga. It might be a good idea to do this first, or we can do it after as well, so I'll approve as-is.

**Review by Xrayez — APPROVED** [review]

**Comment by Calinou** [comment]
This is now superseded by the [Discussions](https://github.com/godotengine/godot-proposals/discussions) tab on this repository, closing. See https://github.com/godotengine/godot-proposals/pull/3978 and https://github.com/godotengine/godot-proposals/pull/3992.

**Pull request closed without merging**
